helpers/controllers/challengesController.ts

```diff
@@ -0,0 +1,60 @@
+import db from '../dbload'
+import { Context } from '../../@types/helpers'
+import _ from 'lodash'
+import { isAdmin } from '../isAdmin'
+
+const { Challenge } = db
```

Comment:
I noticed this pattern in the controller modules and their tests where `db` is imported, and then destructured on a separate line. Is there a reason why we don't just destructure when importing?

Reply:
I don't know, but I think that it looks cleaner if we destructure when importing. So.. i will look at my files again and update as necessary. Thanks!

Reply:
The reason why, is because we are using `export default` in the `dbload.ts` file.
```
export default {
  Lesson,
  Challenge,
  Submission,
  User,
  UserLesson,
  Alert,
  sequelize
}
```
And so it isn't possible to destructure when importing.

Reply:
Thank you for explaining this! So `import { Challenge } from '../dbload'` would fail because the object `default` is exported, but not mentioned in the import, so `Challenge` would not be found.

OK, that brings up another question. I won't delay this PR any further, but I do wonder why we're using the `export default` pattern in `dbload` instead of named exports.

A quick survey shows that we're usually, if not always, importing the default export object from `dbload` and then destructuring it on a different line.

Reply:
I wonder about why we use `export default` in our `dbload` file too. If there isn't a good reason for using `export default`, then I think a refactoring would be nice.

Reply:
refactoring backlog.